alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureMonitorRestApiIT.kt

```diff
@@ -234,6 +296,55 @@ class SecureMonitorRestApiIT : AlertingRestTestCase() {
         assertUserNull(createResponse.asMap()["monitor"] as HashMap<String, Any>)
     }
 
+    fun `test get monitor with an user with get monitor role`() {
+        createUserWithTestDataAndCustomRole(
+            user,
+            TEST_HR_INDEX,
+            TEST_HR_ROLE,
+            TEST_HR_BACKEND_ROLE,
+            getClusterPermissionsFromCustomRole(ALERTING_GET_MONITOR_ACCESS)
+        )
+
+        val monitor = createRandomMonitor(true)
+
+        try {
+            val getMonitorResponse = userClient?.makeRequest(
+                "GET",
+                "$ALERTING_BASE_URI/${monitor.id}",
+                null,
+                BasicHeader(HttpHeaders.CONTENT_TYPE, "application/json")
+            )
+            assertEquals("Get monitor failed", RestStatus.OK, getMonitorResponse?.restStatus())
+        } finally {
+            deleteRoleAndRoleMapping(TEST_HR_ROLE, ALERTING_INDEX_MONITOR_ACCESS)
+        }
+    }
+
+    fun `test get monitor with an user without get monitor role`() {
+        createUserWithTestDataAndCustomRole(
+            user,
+            TEST_HR_INDEX,
+            TEST_HR_ROLE,
+            TEST_HR_BACKEND_ROLE,
+            getClusterPermissionsFromCustomRole(ALERTING_NO_ACCESS_ROLE)
+        )
+
+        val monitor = createRandomMonitor(true)
+
+        try {
+            userClient?.makeRequest(
+                "GET",
+                "$ALERTING_BASE_URI/${monitor.id}",
+                null,
+                BasicHeader(HttpHeaders.CONTENT_TYPE, "application/json")
```

Comment:
Should we add assert fail here, to identify exception did not happen, instead of silently failing the test? Applicable at multiple places.

Reply:
Commented the test for now. Will take it as a part of [this](https://github.com/opensearch-project/alerting/issues/300) issue.